### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Balikuddembe**
In this project,i configured linters, used Figma design to create a UI, used Flexbox to place elements in the page,Used images and backgrounds to enhance the look of this website.
For this milestone ,i have built 2 sections of the mobile website version using the template1 from Figma.The sections include:
- the toolbar 
- the headline section that is right after the header.

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <section id="home_presentation">
+      <h4 class="header1">I'm Mariane
+        Glad to see you!
+      </h4>
+      <p>
+        I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <footer>
```
Comment (CollinsTatang) [inline mobile.html]: - [ ] Please remove this `footer` element from this section.

File: mobile.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section id="home_presentation">
+      <h4 class="header1">I'm Mariane
+        Glad to see you!
+      </h4>
```
Comment (CollinsTatang) [inline mobile.html]: - [ ] Please make use of the `h1 tag` here, since this is the main header of your page

File: mobile.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
```
Comment (CollinsTatang) [inline mobile.css]: - [ ] Kindly change the height attribute value to `height: 618px` and work on the title text font to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
### **Figmal Design**
fig
### **Your Design**
urs

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Balikuddembe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

_Check the comments under the review._
### **Optional**

- [ ] Please rename your file from `mobile.html` to `index.html` and do the same to the CSS file index.css

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Balikuddembe , 

Great work on making the changes requested by a previous reviewer 👏👏👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

 ## Required Changes ♻️

- [ ] As described by the previous reviewer, you need to make sure that all font weights match the Figma design. I have explained this further in the comments below 😀

_Your design:_
image

_Figma Design:_
image

_Check the comments under the review._

## Optional suggestions

- [ ] Kindly remove the spacing around your design 🤗 
  - _This is the default margin of the body._

image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @Balikuddembe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing all the required changes successfully! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**